cfme/configure/access_control.py

```diff
@@ -1435,8 +1367,83 @@ def set_quota(self, **kwargs):
         view.flash.assert_success_message('Quotas for Tenant "{}" were saved'.format(self.name))
         assert view.is_displayed
 
+    def __eq__(self, other):
+        if not isinstance(other, type(self)):
+            return False
+        else:
+            return self.tree_path == other.tree_path
+
+    @property
+    def exists(self):
+        try:
+            navigate_to(self, 'Details')
+            return True
+        except CandidateNotFound:
+            return False
+
+    @cached_property
```

Comment:
**REQUIRED:** I'm not sure this makes sense here. One would assume that a ```tenants``` property on a tenant object would return a list of sub tenants. This doens't do that it actually returns ```self.parent```. If they were intended to be filtered collections, then they should be implemented properly. The same goes for projects below.

Reply:
Corrected